src/RawRabbit/Common/NamingConventions.cs

```diff
@@ -66,13 +67,34 @@ private string GetSubscriberQueueSuffix(Type messageType)
 
 		private string GetApplicationName()
 		{
-			return Directory
-				.GetCurrentDirectory()
-				.Split('\\')
-				.Select(d => d.ToLower())
-				.Last(d => !_disallowedDirectoryNames.Contains(d))
-				.Split('.')
-				.Last();
+			return GetApplicationNameFromUrl(Directory.GetCurrentDirectory());
+		}
+
+		public string GetApplicationNameFromUrl(string url)
+		{
+			var consoleOrIISHostedAppRegex = new Regex(@".*\\(?<ApplicationName>.*[a-zA-Z][^\\]*)(\\[0-9.]*)?\\bin.*");
+			var match = consoleOrIISHostedAppRegex.Match(url);
```

Comment:
According to [https://msdn.microsoft.com/en-us/library/141e06ef%28v=vs.71%29.aspx](Microsoft's guidelines) abbreviations should be in camel case (`consoleOrIisHostedAppRegex`)